scripts/package_building/utils.sh

```diff
@@ -158,3 +158,22 @@ pushd() {
 popd() {
     command popd "$@" > /dev/null
 }
+
+get_branch_from_ini() {
+    git_branch="$1"
+    ini_file="$2"
+    
+    branch=`crudini --get "$ini_file" BRANCHES $git_branch`||true
+    if [[ "$branch" == "" ]];then
+        branch="$git_branch"
+    fi
+    echo $branch
+}
+
+copy_artifacts() {
+    artifacts_folder=$1
+    destination_path=$2
+    
+    cp "$artifacts_folder"/*.rpm "$destination_path" 2> /dev/null || true
```

Comment:
why do you do || true?
what if the copy failed?

Reply:
one of them is supposed to fail , and both cps will fail only if something goes wrong somewhere else in script

Reply:
okay, but don't make everything pass
i would suggest removing || true

and where you use this function and expect a fail, treat it over there.

Reply:
I suggest making a test before, if the files exist and then copy them over